### PR TITLE
docs/charra-concept-on-pc.md add doumentation

### PR DESCRIPTION
**Pull request by 0xDen**
Signed-off-by: Denis Semakin <insane79@mail.ru>

### CODE REVIEW

File: docs/charra-concept-on-pc.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# How to run CHARRA concept on PC
+
+The detailed steps of how to build and run CHARRA applications (Attester and Verifier) are
+discribed on corresponding GitHub repository:
```
Comment (arturkow2000) [inline docs/charra-concept-on-pc.md:4]:
```suggestion
described on corresponding GitHub repository:
```

File: docs/charra-concept-on-pc.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+The first path is faster and more preferable. When the project is built one may run and test
+Attester and Verifier application:
+1. Configure network for docker in order to allow communicate containers.
+E.g. Edit file ./docker/run.sh and add internal virtual network:
+```
+--network=charra_default
+```
+to docker run command
+2. Specify IP address of Attester in file src/verifier.c (char dst_host[16] in src/verifier.c)
```
[the remark this answers is not in the document]
Comment (arturkow2000) [reply docs/charra-concept-on-pc.md:19]:
```suggestion

2. Specify IP address of Attester in file src/verifier.c (char dst_host[16] in src/verifier.c)
```

File: docs/charra-concept-on-pc.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# How to run CHARRA concept on PC
+
+The detailed steps of how to build and run CHARRA applications (Attester and Verifier) are
+discribed on corresponding GitHub repository:
+ - [Build and run CHARRA on PC](https://github.com/Fraunhofer-SIT/charra#build-and-run)
```
Comment (arturkow2000) [inline docs/charra-concept-on-pc.md:5]: I think you should briefly describe the exact steps you used to build Charra container.
Theirs documentation isn't 100% clear, e.g. documentation says to tag the TPM emulator container `tpm2software/tpm2-tss:ubuntu-20.04` but I had to tag it `ghcr.io/tpm2-software/ubuntu-20.04:latest` to build Charra container.

File: docs/charra-concept-on-pc.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+In general there are two common ways to reach the goal:
+* Build docker container with binary packages
+* Build neccessary packages from sources inside docker container
+
+The first path is faster and more preferable. When the project is built one may run and test
+Attester and Verifier application:
+1. Configure network for docker in order to allow communicate containers.
+E.g. Edit file ./docker/run.sh and add internal virtual network:
+```
+--network=charra_default
```
Comment (arturkow2000) [inline docs/charra-concept-on-pc.md:16]: How to create this network?

**Review by arturkow2000 — CHANGES_REQUESTED** [review]
@0xDen I left a review.

Besides these few comments you can, and even should spell check the document. You can ask @macpijan for access to Grammarly.

File: docs/charra-concept-on-pc.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (0xDen) [reply docs/charra-concept-on-pc.md:20]: Fixed

File: docs/charra-concept-on-pc.md (hunk 1 above)
Comment (0xDen) [reply docs/charra-concept-on-pc.md]: Fixed

File: docs/charra-concept-on-pc.md (hunk 4 above)
Comment (0xDen) [reply docs/charra-concept-on-pc.md:16]: After I run `./docker/build.sh` script I have found this network... 
```shell
$ docker network ls
NETWORK ID     NAME             DRIVER    SCOPE
....
a13b7967a1d4   charra_default   bridge    local
....
```
And just specify its name in script `./docker/run.sh`

File: docs/charra-concept-on-pc.md (hunk 3 above)
Comment (0xDen) [reply docs/charra-concept-on-pc.md:5]: Hm... just run `./docker/build.sh` and necessary packages were installed in container

File: docs/charra-concept-on-pc.md (hunk 4 above)
Comment (0xDen) [reply docs/charra-concept-on-pc.md:26]: Fixed

**Comment by 0xDen** [comment]
@arturkow2000 Thanks for remarks. Please review again

**Pull request merged**